addons/hr_work_entry_contract/wizard/hr_work_entry_regeneration_wizard.py

```diff
@@ -14,13 +15,19 @@ class HrWorkEntryRegenerationWizard(models.TransientModel):
     latest_available_date = fields.Date('Latest date', compute='_compute_latest_available_date')
     latest_available_date_message = fields.Char(readonly=True, store=False, default='')
     date_from = fields.Date('From', required=True, default=lambda self: self.env.context.get('date_start'))
-    date_to = fields.Date('To', required=True, default=lambda self: self.env.context.get('date_end'))
-    employee_id = fields.Many2one('hr.employee', 'Employee', required=True)
+    date_to = fields.Date('To', required=True, compute='_compute_date_to', store=True,
+            readonly=False, default=lambda self: self.env.context.get('date_end'))
+    employee_id = fields.Many2one('hr.employee', 'Employee', default=lambda self: self.env.context.get('employee_id'), required=True)
```

Comment:
If you use the context key `default_employee_id` you can get rid of the default= value here